examples/2d/plot_scattering_disk.py

```diff
@@ -1,102 +1,191 @@
+# Observe how the amplitude of first-order coefficients is strongest along the direction of edges, and that they
+# exhibit by construction a central symmetry.
 
-fig,ax = plt.subplots()
+# Define figure size and grid on which to plot input digit image, first-order and second-order scattering coefficients
+fig = plt.figure(figsize=(47, 15))
+spec = fig.add_gridspec(ncols=3, nrows=1)
+
+gs = gridspec.GridSpec(1, 3, wspace=0.1)
+gs_order_1 = gridspec.GridSpecFromSubplotSpec(window_rows, window_columns, subplot_spec=gs[1])
+gs_order_2 = gridspec.GridSpecFromSubplotSpec(window_rows, window_columns, subplot_spec=gs[2])
+
+# Start by plotting input digit image and invert colors
+ax = plt.subplot(gs[0])
+ax.set_xticks([])
+ax.set_yticks([])
+ax.imshow(255 - src_img, cmap='gray', interpolation='nearest', aspect='auto')
+
+# Plot first-order scattering coefficients
+ax = plt.subplot(gs[1])
+ax.set_xticks([])
+ax.set_yticks([])
+
+l_offset = int(L - L / 2 - 1)  # follow same ordering as Kymatio for angles
 
-plt.imshow(1-src_img,cmap='gray',interpolation='nearest', aspect='auto')
-ax.axis('off')
-offset = 0.1
 for row in range(window_rows):
     for column in range(window_columns):
-        ax=fig.add_subplot(window_rows, window_columns, 1 + column + row * window_rows, projection='polar')
-        ax.set_ylim(0, 1)
+        ax = fig.add_subplot(gs_order_1[row, column], projection='polar')
         ax.axis('off')
-        ax.set_yticklabels([])  # turn off radial tick labels (yticks)
-        ax.set_xticklabels([])  # turn off degrees
-        # ax.set_theta_zero_location('N')  # 0° to North
-        coefficients = scattering_coefficients[:, row, column]
+        coefficients = scat_coeffs_order_1[:, row, column]
         for j in range(J):
             for l in range(L):
-                coeff = coefficients[l + (J - 1 - j) * L]
-                color = mpl.colors.to_hex(mapper.to_rgba(coeff))
-                ax.bar(x=(4.5+l) *  np.pi / L,
-                       height=2*(2**(j-1) / 2**J),
-                       width=2 * np.pi / L,
-                       bottom=offset + (2**j / 2**J) ,
+                coeff = coefficients[l + j * L]
+                color = mapper_order_1.to_rgba(coeff)
+                angle = (l_offset - l) * np.pi / L
+                radius = 2 ** (-j - 1)
+                ax.bar(x=angle,
+                       height=radius,
+                       width=np.pi / L,
+                       bottom=radius,
                        color=color)
-                ax.bar(x=(4.5+l+L) * np.pi / L,
-                       height=2*(2**(j-1) / 2**J),
-                       width=2 * np.pi / L,
-                       bottom=offset + (2**j / 2**J) ,
+                ax.bar(x=angle + np.pi,
+                       height=radius,
+                       width=np.pi / L,
+                       bottom=radius,
                        color=color)
+
+# Plot second-order scattering coefficients
```

Comment:
what about make it a title?

Reply:
This is in fact a comment within a Jupyter cell so I am not sure how it would apply